### DISCUSSION TITLE
Release info

### DISCUSSION THREAD

**Discussion by marozman2** [ovh/cds#6496] [Q&A]
Hi,
I think the roadmap info is broken, can you please fix it, thanks! I would like to know when the next official release is planned.
[https://github.com/ovh/cds/blob/master/ROADMAP.md]

Additionally I would like to know, whether you can confirm that this commit and any dependent ones will be part of the next official release (or not)? Thanks
[https://github.com/ovh/cds/pull/6407]
Marton

**Comment by jptiancai** [comment]
I found it in [this](https://github.com/ovh/cds/blob/24bca9e73a44a4b0cd4ebd1c5d16daa1444d4082/ROADMAP.md)
and will add it with [pr](https://github.com/ovh/cds/pull/6550)

**Comment by yesnault** [comment]
We will prepare an updated Roadmap page. We are indeed on the development of the future of cds with all things handled ascode. It's a big project that will last several months, with a new ui and an updated workflow model (without loss of functionality, but a simplification to come). 
When we have reached a first level on this ascode feature, we will decide to make a release (this should not delay any longer).

**Reply by marozman2** [reply]
Great to hear!
